stories/titleAndListWithOptionalLogos.stories.js

```diff
@@ -4,16 +4,23 @@ import Theme from './theme'
 import BlueBackground from '../src/components/Common/BlueBackground'
 import GreyBackground from '../src/components/Common/GreyBackground'
 import { Grid } from '../src/components/grid'
-import TitleAndList from '../src/components/Common/TitleAndList'
+import TitleAndList from '../src/components/Common/TitleAndListWithOptionalLogos'
 import StyledLink from '../src/components/Common/StyledLink'
 
 addDecorator(Theme)
 
+const ExampleImage = {
+  title: 'Community Support Logo',
+  file: { url: '/community_support.svg' },
```

Comment:
'./assets/community_support.svg' ?

It might be nice to import at the top of the file & then pass in here like:
`import CommunitySupportLogo from './assets/community_support.svg'`
then
```
const ExampleImage = {
  title: 'Community Support Logo',
  file: { url: CommunitySupportLogo }
}
```

Does that work?

Reply:
I don't think that would work because the way storybook works, is that you give it a path on build (it was missing in package.json, that's why your screenshot above doesn't have them), and the path is `./stories/assets`. Storybook knows to automatically look into that folder when given a link.

Reply:
Nah. My misunderstanding.

Thanks!

I'm not sure why your images are broken on storybook then?